projects/online/online/utils/gdb.py

```diff
@@ -111,6 +111,9 @@ def submit_low_latency_pe(
         )
         logging.debug("Skymap submitted")
 
+        # rename so we can later write kde file
+        skymap_fname.rename(event_dir / "amplfi.multiorder.fits,0")
```

Comment:
You don't have to rename this. GraceDb will automatically version the second upload to add a `,1` for the same filename.

Reply:
This is after the gracedb upload

Reply:
Thanks for clarifying. Makes sense.